components/kyma-environment-broker/common/orchestration/dto.go

```diff
@@ -104,15 +109,27 @@ type TargetSpec struct {
 	Exclude []RuntimeTarget `json:"exclude,omitempty"`
 }
 
+type KymaDetailResponse struct {
```

Comment:
I would reuse the `Paramaters.Kyma` and `Parameters.Kubernetes` for sending kyma and kubernetes info in the response, and wouldn't introduce new structs for the response